etna/insights/models.py

```diff
@@ -30,14 +32,25 @@ def get_context(self, request):
 
     content_panels = BasePage.content_panels + [
         FieldPanel("sub_heading"),
-        PageChooserPanel("featured_insight"),
```

Comment:
This was added recently by myself for another ticket, and will need adding back in.

Reply:
Yes, sorry for this, It was missed out during the merge, which had a few conflicts.